values_local_backend.yaml

```diff
@@ -0,0 +1,7 @@
+options:
```

Comment:
To keep with the current naming (`values` / `local-values`) you could name this file `local-backend-values.yaml`.